README.md

```diff
@@ -65,8 +65,8 @@ The `onChange` action is called with two arguments:
 export default class MyRoute extends Route {
   @action
   reorderItems(itemModels, draggedModel) {
```

Comment:
would you also be willing to update the example? 
 - currentModel is ???
 - the template above references `this.reorderItems`, yet this is on a route (we should probably change everything to be in the form of a component demo / example